mlir/test/Transforms/print-op-graph-cycles.mlir

```diff
@@ -4,41 +4,41 @@
 //       DFG:   compound = true;
 //       DFG:   subgraph cluster_1 {
 //       DFG:     v2 [label = " ", shape = plain];
-//       DFG:     label = "builtin.module : ()\n";
+//       DFG:     label = "builtin.module : ()\l";
 //       DFG:     subgraph cluster_3 {
 //       DFG:       v4 [label = " ", shape = plain];
 //       DFG:       label = "";
 //       DFG:       subgraph cluster_5 {
 //       DFG:         v6 [label = " ", shape = plain];
-//       DFG:         label = "test.graph_region : ()\n";
+//       DFG:         label = "test.graph_region : ()\l";
```

Comment:
Better to use some `-NEXT` whenever applicable. Without -NEXT, if there is a change, the FileCheck suggestion would look strange.

Reply:
https://github.com/llvm/llvm-project/pull/125509/commits/7a2f24a7493d00db258eb363f10fc2344e00539e